js/apps/events-page/directives/loadMore.js

```diff
@@ -0,0 +1,18 @@
+((app) => {
+	const loadMoreDirective = (CONSTANTS) => {
+		const directive = {
+			restrict: 'E',
+			templateUrl: CONSTANTS.templateUrls.loadMore,
+			scope: {
+				loadNextPage: '=',
+				chunkSize: '='
```

Comment:
```chunk``` seems misleading to me somethign like numberOfResults seems more meaningful. This is also no longer required?

Reply:
Removed!